examples/connect-app/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```

Comment:
this example should be a bit more substantive - maybe print the data we received after pressing a button to retrieve data?